### PR TITLE
fix typos, rework explanations

### PR DESCRIPTION
**Pull request by hboutemy**
little wording improvements from a first-time reader

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Scan your libraries against either the free [OSS Index](https://ossindex.sonatyp
 ## Features
 
 - Scan npm, Maven, RubyGems, Go (`dep` and `go mod`), R (see known quirks) or PyPi projects (Go is only supported on Linux or OS/X)
-- See all components with vulnerable components highlighted
+- See all all components, with vulnerable ones highlighted
```
Comment (bhamail) [inline README.md:10]: Duplicate "all"?

**Review by bhamail — APPROVED** [review]
LGTM. One minor comment.

File: README.md (hunk 1 above)
Comment (hboutemy) [reply README.md]: fixed, thank you

**Pull request merged**